src/wgps/events.ts

```diff
@@ -0,0 +1,11 @@
+export class WgpsStatusEvent
+  extends CustomEvent<{ todo: number; all: number }> {
+  constructor(todo: number, all: number) {
+    super("status", {
+      detail: {
+        todo,
```

Comment:
Nitpick: could we change this to `remaining`, as at least for me `todo` triggers thoughts of 'oh, I guess this is something someone forgot to do'?